zap-extensions.properties

```diff
@@ -0,0 +1,3 @@
+path.variable.maven_repository=C\:\\Users\\acaetano\\.m2\\repository
```

Comment:
This file should be removed? (same for zap-extensions.xml)

Reply:
ok, removed